src/components/Navigation/Navigation.tsx

```diff
@@ -1,13 +1,14 @@
 import React, { useState } from "react";
 import classnames from "classnames";
 import { graphql, Link, useStaticQuery } from "gatsby";
+import { AnimatePresence, motion } from "framer-motion";
 import { Container } from "@/components/Container";
 import { Logo } from "@/components/Logo";
 import { Icon } from "@/components/Icon";
 import { Breakpoints, useResizeObserver } from "@/hooks/useResizeObserver";
-import { AnimatePresence, motion } from "framer-motion";
 
 import { NavigationItemFragment } from "@/graphql/graphql-types";
+import { AnthemLogo } from "../AnthemLogo/AnthemLogo";
```

Comment:
use the alias path `@/components/AnthemLogo` whenever possible instead of relative paths